frontend/src/container/PublicProfile.jsx

```diff
@@ -384,6 +431,15 @@ export class PublicProfile extends React.Component {
 
   handleCloseUploadPopup = () => this.setState({ displayUploadPopup: undefined })
 
+  handleClosePopup = () => {
+    this.setState({ userCall: undefined })
+  }
+
+  openCallWindow = () => {
+    const { state } = this
+    window.open(state.userCall.url)
+  }
+
```

Comment:
Is this used?

Reply:
Actually, it is not, and I will remove it

Reply:
So : It was not used, but I was needed, but it had the wrong naming so I fixed this